networkx/algorithms/tests/test_dominance.py

```diff
@@ -261,3 +261,37 @@ def test_loops_larger(self):
                   'exit': set()}
         for n in df:
             assert_equal(set(df[n]),set(answer[n]))
+
+
+class TestDominatorTree(object):
+
+    def test_dominator_tree(self):
+        # from page 5 (labeled as 388 in bottom right)
+        # http://pages.cs.wisc.edu/~fischer/cs701.f08/lectures/Lecture19.pdf
+        edges = [(1, 2), (2, 3), (2, 4), (3, 5), (4, 5), (5, 6), (6, 7),
+                (7, 6), (7, 8)]
+        G = nx.DiGraph(edges)
+        T = nx.dominator_tree(G, 1)
+        assert_equal(sorted(list(T.edges())),
```

Comment:
There is a `networkx.testing.utils.assert_edges_equal` function so that you don't have to sort these things. Something like

```python
from networkx.testing.utils import assert_edges_equal

assert_edges_equal(T.edges(), expected_edges)
```